commands/config.go

```diff
@@ -119,6 +122,10 @@ func (cmd *configCommand) args() (res []string, switchedToTLS bool) {
 		res = append(res, fmt.Sprintf("--paths-base=%s", cmd.BasePath))
 	}
 
+	if GlobalFlags.PMMAgentListenPort != 0 && GlobalFlags.PMMAgentListenPort != agentlocal.DefaultPMMAgentListenPort {
+		res = append(res, fmt.Sprintf("--listen-port=%d", GlobalFlags.PMMAgentListenPort))
+	}
+
```

Comment:
let's put it above where we set global flags and let's remove check that listen port is not equal to default one.

Reply:
Moved above. Check if listen port is default removed.